### PR TITLE
MC test batch - Printer standard

### PR DESCRIPTION
**Pull request by omercier**
## Description

Great reboot in progress.

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [x] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

REFS: MON-23834

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md [hunk 1]
```diff
@@ -2,60 +2,489 @@
...
+
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
+
+<Tabs groupId="sync">
+<TabItem value="HW-Printer-standard-rfc3805-custom" label="HW-Printer-standard-rfc3805-custom">
+
+| Alias              | Modèle de service                                          | Description                                                                                        |
+|:-------------------|:-----------------------------------------------------------|:---------------------------------------------------------------------------------------------------|
+| Cover-Status       | HW-Printer-Standard-Rfc3805-CoverStatus-SNMP-custom        | ContrÃ                                                                                             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md:24]: ContrÃ?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md [hunk 2]
```diff
@@ -2,60 +2,489 @@
...
+| Cover-Status       | HW-Printer-CoverStatus-Hardware-standard-SNMP-custom       | Contrôle les états des éléments recensés dans la table "cover" de la MIB définie dans la rfc3805   |
+| Impressions        | HW-Printer-Standard-Rfc3805-Impressions-SNMP-custom        | ContrÃ                                                                                             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md:26]: ContrÃ?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md [hunk 3]
```diff
@@ -2,60 +2,489 @@
...
+| Impressions        | HW-Printer-Impressions-Usage-standard-SNMP-custom          | Contrôle le nombre d'impressions entre deux contrôles                                              |
+| MarkerSupply-Usage | HW-Printer-Standard-Rfc3805-Markersupply-Usage-SNMP-custom | ContrÃ                                                                                             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md:28]: ContrÃ?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md [hunk 4]
```diff
@@ -2,60 +2,489 @@
...
+| MarkerSupply-Usage | HW-Printer-Markersupply-Usage-standard-SNMP-custom         | Contrôle le niveau d'encre des cartouches                                                          |
+| PaperTray-Usage    | HW-Printer-Standard-Rfc3805-PaperTray-Usage-SNMP-custom    | ContrÃ                                                                                             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md:30]: ContrÃ?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md [hunk 5]
```diff
@@ -2,60 +2,489 @@
...
+| PaperTray-Usage    | HW-Printer-PaperTray-Usage-standard-SNMP-custom            | Contrôle le taux de remplissage des différents bacs à papier                                       |
+| Printer-Errors     | HW-Printer-Standard-Rfc3805-Errors-SNMP-custom             | ContrÃ                                                                                             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md:32]: ContrÃ?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md [hunk 6]
```diff
@@ -2,60 +2,489 @@
...
+| Printer-Errors     | HW-Printer-Errors-standard-SNMP-custom                     | Contrôle des erreurs imprimantes liées au papier                                                   |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md:33]:
```suggestion
| Printer-Errors     | HW-Printer-Errors-standard-SNMP-custom                     | Contrôle des erreurs imprimante liées au papier                                                   |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md [hunk 7]
```diff
@@ -2,60 +2,489 @@
...
+| Printer-Hardware   | HW-Printer-Standard-Rfc3805-Hardware-SNMP-custom           | Controle diffÃ                                                                                     |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md:34]: 1. Contrôle
2. diffÃ?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md [hunk 8]
```diff
@@ -2,60 +2,489 @@
...
+| Printer-Hardware   | HW-Printer-Hardware-standard-SNMP-custom                   | Controle différents indicateurs matériels sur une imprimante                                       |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md:35]:
```suggestion
| Printer-Hardware   | HW-Printer-Hardware-standard-SNMP-custom                   | Contrôle différents indicateurs matériels sur une imprimante                                       |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md [hunk 9]
```diff
@@ -2,60 +2,489 @@
...
+</TabItem>
+</Tabs>
+
+### Règles de découverte
+
+#### Découverte d'hôtes
+
+| Nom de la règle | Description                                                                                                                                                                                                                                    |
+|:----------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SNMP Agents     | Discover your resource through a SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) to get the discovery rule and create a template mapper for the **HW-Printer-standard-rfc3805-custom** host template |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md:48]:
```suggestion
| SNMP Agents     | Discover your resource through a SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) connector to get the discovery rule and create a template mapper for the **HW-Printer-standard-rfc3805-custom** host template |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md [hunk 10]
```diff
@@ -2,60 +2,489 @@
...
+| *printer*~*errors*#status | N/A   |
+
+> Concerne les modèles de service suivants : Printer-Errors, Printer-Errors
+
+> Pour obtenir ce nouveau format de métrique, incluez la valeur **--use-new-perfdata** dans la macro de service **EXTRAOPTIONS**.
+
+</TabItem>
+<TabItem value="Printer-Hardware*" label="Printer-Hardware*">
+
+Pas de métrique pour ce mode.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md:103]:
```suggestion
Pas de métrique pour ce service.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md [hunk 11]
```diff
@@ -2,60 +2,489 @@
...
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+### Configuration SNMP
+
+Afin de superviser votre ressource en SNMP,  il est nécessaire de configurer l'agent SNMP
+sur la ressource comme indiqué sur la documentation officielle du constructeur.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md:113]:
```suggestion
sur la ressource comme indiqué dans la documentation officielle du constructeur.
```

File: pp/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md [hunk 12]
```diff
@@ -2,60 +2,488 @@
...
+
+</TabItem>
+</Tabs>
 
 ## Prerequisites
 
-### Centreon Plugin
+### SNMP Configuration
+
+To use this pack, the SNMP service must be properly configured on your ressource.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md:111]:
```suggestion
To use this pack, the SNMP service must be properly configured on your resource.
```

File: pp/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md [hunk 13]
```diff
@@ -2,60 +2,488 @@
...
+| *printer*~*errors*#status | N/A   |
+
+> Applies to the following service templates: Printer-Errors, Printer-Errors
+
+> To obtain this new metric format, include **--use-new-perfdata** in the **EXTRAOPTIONS** service macro.
+
+</TabItem>
+<TabItem value="Printer-Hardware*" label="Printer-Hardware*">
+
+No metrics ofr this mode.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md:102]:
```suggestion
No metrics for this service.
```

File: pp/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md [hunk 14]
```diff
@@ -2,60 +2,488 @@
...
+</TabItem>
+</Tabs>
+
+### Discovery rules
+
+#### Host discovery
+
+| Rule name       | Description                                                                                                                                                                                                                                    |
+|:----------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SNMP Agents     | Discover your resource through a SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) to get the discovery rule and create a template mapper for the **HW-Printer-standard-rfc3805-custom** host template |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md:47]:
```suggestion
| SNMP Agents     | Discover your resource through a SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) connector to get the discovery rule and create a template mapper for the **HW-Printer-standard-rfc3805-custom** host template |
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md (hunk 1 above)
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md:24]:
```suggestion
| Cover-Status       | HW-Printer-Standard-Rfc3805-CoverStatus-SNMP-custom        | Contrôle les états des éléments recensés dans la table "cover" de la MIB définie dans la rfc3805.                                                                                             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md (hunk 2 above)
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md:26]:
```suggestion
| Impressions        | HW-Printer-Standard-Rfc3805-Impressions-SNMP-custom        | Contrôle le nombre d'impressions entre deux contrôles.                                                                                             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md (hunk 3 above)
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md:28]:
```suggestion
| MarkerSupply-Usage | HW-Printer-Standard-Rfc3805-Markersupply-Usage-SNMP-custom | Contrôle le niveau d'encre des cartouches.                                                                                             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md (hunk 4 above)
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md:30]:
```suggestion
| PaperTray-Usage    | HW-Printer-Standard-Rfc3805-PaperTray-Usage-SNMP-custom    | Contrôle le taux de remplissage des différents bacs à papier.                                                                                             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md (hunk 5 above)
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md:32]:
```suggestion
| Printer-Errors     | HW-Printer-Standard-Rfc3805-Errors-SNMP-custom             | Contrôle des erreurs imprimante liées au papier (bourrages, etc).                                                                                             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md (hunk 7 above)
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-printers-standard-rfc3805-snmp.md:34]:
```suggestion
| Printer-Hardware   | HW-Printer-Standard-Rfc3805-Hardware-SNMP-custom           | Contrôle différents indicateurs matériels sur une imprimante.                                                                                     |
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**